test/unit/test_helpers.py

```diff
@@ -8,6 +8,10 @@
 import pytest
 
 from pureport_client import helpers
+from pureport import models
+from unittest.mock import patch, Mock
+import os
+import json
```

Comment:
I think this import ordering is incorrect...

Reply:
sure, I'll fix that and update the PR

> You may need to run `flake8` on this.

I think `flake8` runs as part of tox and in the workflow that runs on the pull request, but it's possible the rules aren't as strict as they should be